app/models/job_execution.rb

```diff
@@ -29,6 +29,7 @@ def initialize(reference, job, env: {}, output: OutputBuffer.new, &block)
     @repository.executor = @executor
 
     on_finish do
+      Rails.logger.info("Calling finish callback for Job Execution #{id}")
```

Comment:
might be easier to read/parse when using a consistent prefix like `JobExecution#on_finish`